packages/sdk-themes/src/themes/monite.ts

```diff
@@ -6,12 +6,14 @@ import type {
 } from '@mui/material/styles/createPalette.js';
 import type { Theme, ThemeOptions } from '@mui/material/styles/createTheme.js';
 import type { TypographyOptions } from '@mui/material/styles/createTypography.js';
-import type {} from '@mui/x-data-grid/themeAugmentation';
+
+import { neutralsTransparent, primary } from '../colors/monite/index.js';
```

Comment:
If you don't mind, let's maintain `colors` in the same file with the theme. I'm not sure that we will share the palette.